memcache/memcache.go

```diff
@@ -711,3 +727,160 @@ func (c *Client) incrDecr(verb, key string, delta uint64) (uint64, error) {
 	})
 	return val, err
 }
+
+type MetaSetItem struct {
+	Key    string
+	Flags  MetaSetFlags
+	Length int32
+	Value  []byte
+}
+
+type MetaSetFlags struct {
+	// use if the key provided is base 64 encoded
+	// equivalent to the b flag
+	IsKeyBase64 bool
```

Comment:
I know both of you are working on it at the same time, but I wonder if we can make a common struct for the common flags between set and get and then embed it inside each struct? @spaiStrp @alexchiriac-stripe